frontend/src/app/supportAdmin/ManageUsers/AdminManageUser.test.tsx

```diff
@@ -173,28 +171,21 @@ describe("Admin manage users", () => {
       expect(document.body).toMatchSnapshot();
     });
     it("displays user is an admin", async () => {
-      const { user } = renderComponent([
-        {
-          request: {
-            query: FindUserByEmailDocument,
-            variables: { email: "admin@example.com" },
-          },
-          result: {
-            data: null,
-            errors: [
-              {
-                message:
-                  "header: Error finding user email; body: Please escalate this issue to the SimpleReport team.",
-                locations: [],
-                path: ["user"],
-                extensions: {
-                  classification: "ExecutionAborted",
```

Comment:
Were these extensions used in the past? It looks like we weren't doing any assertions on it so I'm assuming it's all good to clean up

Reply:
Yeah Typescript was telling me to use the `GraphQLError` type that I added on 182 and since we just assert by the error message, didn't bother to port over the other pieces.